site/content/_shared/cli-env-vars.md

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Could you add `sitemapExclude` here to not leak this partial?

And maybe we could unify with the wording we have here? https://github.com/checkly/docs.checklyhq.com/blob/main/site/content/_shared/test-session-details.md